src/liquid/components/ld-bg-cells/assets/hexagon-cell.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="3884" height="2861" viewBox="0 0 3884 2861" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think we can optimize those svg files manually in order to make them load quicker.

Reply:
We'll fix that with the icon PR.